### PR TITLE
Explicit dependencies (from satreix)

### PR DESCRIPTION
**Pull request by nitnelave**
Rebase and re-work of #7

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+guessit = "*"
+pyinotify = "*"
+pysubs2 = "*"
+pyyaml = "*"
+requests = "*"
+subliminal = "*"
+tmdbsimple = "*"
+whoosh = "*"
+reorder-python-imports = "*"
+yapf = "*"
```
Comment (satreix) [inline Pipfile]: reorder-python-imports and yapf should not appear in packages.

File: search/crawler.py [hunk 2]
```diff
@@ -1,13 +1,13 @@
-#! /usr/bin/env python
-
-import requests
+#!/usr/bin/env python3
```
Comment (satreix) [inline search/crawler.py]: Shebang are of no use in file that do not have a main and are not executable.

File: search/search.py [hunk 3]
```diff
@@ -1,30 +1,27 @@
 #! /usr/bin/env python
```
Comment (satreix) [inline search/search.py]: Use the python3 shebang.

File: subs/opensubs.py [hunk 4]
```diff
@@ -1,19 +1,23 @@
 #! /usr/bin/env python
```
Comment (satreix) [inline subs/opensubs.py]: Ditto https://github.com/nitnelave/tolmer.fr/pull/12/files#r332075403

File: subs/shift.py [hunk 5]
```diff
@@ -1,13 +1,12 @@
 #! /usr/bin/env python3
```
Comment (satreix) [inline subs/shift.py]: Ditto https://github.com/nitnelave/tolmer.fr/pull/12/files#r332075403

File: subs/sub2srt.py [hunk 6]
```diff
@@ -1,75 +1,62 @@
-#!/usr/bin/env python3
```
Comment (satreix) [inline subs/sub2srt.py]: Please use the python3 shebang.

File: Pipfile (hunk 1 above)
Comment (nitnelave) [reply Pipfile]: Done

File: tools/format.sh [hunk 7]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+find . -type f -name '*.py' -print0 |
+  xargs -0 pipenv run reorder-python-imports \
+    --exit-zero-even-if-changed \
+    --py37-plus \
+    --separate-relative
+find . -name '*.py' -print0 | xargs -0 pipenv run yapf -i
```
Comment (nitnelave) [inline tools/format.sh]: I changed that from `yafp -r` since it took forever on my machine, whereas this is sub-second.

**Pull request merged**